BeatThat_Basics.js

```diff
@@ -0,0 +1,74 @@
+var WAITPLAYER1 = "waiting for Player 1";
```

Comment:
Nice use of variables to store string constants. One way to enhance it would be to use scream case, which is the convention. (i.e. WAIT_PLAYER_1)